features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -688,6 +688,8 @@
         <RedirectToRequestedRedirectUri>false</RedirectToRequestedRedirectUri>
         {% endif %}
 
+        <PublishPasswordGrantLogin>{{publish_password_grant_auth_login.enable}}</PublishPasswordGrantLogin>
```

Comment:
The config name doesn't match our convention. It should be hierarchical separated by dots.

Reply:
Changed.